File: app/src/main/java/com/example/android/popularmovies/database/PosterUriConverter.java
package com.example.android.popularmovies.database;

import android.net.Uri;

import androidx.room.TypeConverter;

class PosterUriConverter {

    @TypeConverter
    public static String toString(Uri uri){
        return uri.toString();
    }

    @TypeConverter
    public static Uri toUri(String str){
        return Uri.parse(str);
    }

}

File: app/src/main/java/com/example/android/popularmovies/database/PosterUriConverter.kt
package com.example.android.popularmovies.database

import android.net.Uri
import androidx.room.TypeConverter

class PosterUriConverter {
    @TypeConverter
    fun toString(uri: Uri): String {
        return uri.toString()
    }

    @TypeConverter
    fun toUri(str: String?): Uri {
        return Uri.parse(str)
    }
}